sympy/polys/tests/test_numberfields.py

```diff
@@ -637,9 +637,9 @@ def test_AlgebraicNumber():
 
     a = AlgebraicNumber(sqrt(2))
     b = to_number_field(sqrt(2))
-    assert a.args == b.args == (sqrt(2), Tuple())
+    assert a.args == b.args == (sqrt(2), Tuple(1, 0))
```

Comment:
> fixing it here

Do you mean defining ``None`` as the default? (It is not connected to printing.)

> why is that different from (0,0)?

Actually, it is not. It is just the "reduced representation" of the zero polynomial. The tuple (1, 0) represents the linear polynomial ``X``. There could also be polynomials of higher degree (i.e. tuples with more than two elements). They are essential when considering elements of algebraic number fields. But this area is currently in unfinished state.